fhir-persistence-schema/README.md

```diff
@@ -4,20 +4,18 @@ Builds and manages the multi-tenant FHIR R4 RDBMS schema (Db2). Includes Derby s
 
 # To Execute the fhir-persistence-schema 
```

Comment:
I decided to keep the VERSION specifier on the persistence-schema jar, so please revert all the changes to this file.  Otherwise, I'll just revert as part of https://github.com/IBM/FHIR/pull/587

Reply:
I reverted to master